stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/StorageHelper.kt

```diff
@@ -178,7 +185,7 @@ private fun AttachmentMetaData.getTitleWithExtension(): String {
     return if (extension.isNullOrEmpty() && !mimeType.isNullOrEmpty()) {
         "$newTitle.${MimeTypeMap.getSingleton().getExtensionFromMimeType(mimeType)}"
     } else {
-        // TODO: Attachment's title should never be null. Review AttachmentMetaData class
+        // Attachment's title should never be null. Review AttachmentMetaData class
```

Comment:
Should we just remove the TODO? It was there for a reason - can we create a new GH ticket to make sure this is not lost?

Reply:
Let me check it, I will create a GH Ticket if it is needed